lib/create-esbuild-script.js

```diff
@@ -22,27 +22,39 @@ async function createEsbuildScript(answers) {
     format: "esm",
     ...sharedOptions,
   });`;
-  const dualBuildOptions = `  await Promise.all([
-  // Build a CJS Node.js bundle
-  esbuild.build({
+  const dualBuildOptions = `  await esbuild.build({
     entryPoints,
-    outdir: "pkg/dist-node",
+    outdir: "pkg/dist-bundle",
     bundle: true,
-    platform: "node",
-    target: "node18",
-    format: "cjs",
-    ...sharedOptions,
-  }),
-  // Build an ESM browser bundle
-  esbuild.build({
-    entryPoints,
-    outdir: "pkg/dist-web",
-    bundle: true,
-    platform: "browser",
+    platform: "neutral",
     format: "esm",
     ...sharedOptions,
   }),
 ]);`;
+  const nodeJSExports = `exports: {
+  ".": {
+    types: "./dist-types/index.d.ts",
+    import: "./dist-node/index.js",
+    // Tooling currently are having issues with the "exports" field when there is no "default", ex: TypeScript, eslint
+    default: "./dist-node/index.js",
+  },
+}`;
+  const browserExports = `exports: {
+  ".": {
+    types: "./dist-types/index.d.ts",
+    import: "./dist-browser/index.js",
+    // Tooling currently are having issues with the "exports" field when there is no "default", ex: TypeScript, eslint
+    default: "./dist-browser/index.js",
+  },
+}`;
+  const dualExports = `exports: {
+  ".": {
+    types: "./dist-types/index.d.ts",
+    import: "./dist-bundle/index.js",
+    // Tooling currently are having issues with the "exports" field when there is no "default", ex: TypeScript, eslint
```

Comment:
Will this give problems with `package.json`? Are comments supported?

Reply:
That's only the ESBuild script. The comments don't  get outputted to the final `package.json`